src/components/Notification/Notification.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { Toast, ToastHeader, ToastBody, Button } from 'reactstrap'
+import Icon from '../Icon/Icon'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+const propTypes = {
+  header: PropTypes.string,
+  content: PropTypes.string,
+  state: PropTypes.string,
+  dismissable: PropTypes.bool,
+  fix: PropTypes.string,
+  withIcon: PropTypes.bool
+}
+
+// custom style
+const NotificationStyle = {
```

Comment:
Because this is a Storybook issue, would it be possible to have a `wrapperStyle` prop here and passing this from the Storybook context?

Reply:
Sure, but there's a slight problem with this. The user will have to give a wrapper style each time they render the component. Otherwise, it get's place at bottom right corner due to BI css.

Reply:
I'll get this done in my new commit though.